nautobot/virtualization/forms.py

```diff
@@ -242,6 +243,7 @@ class Meta:
         widgets = {
             "primary_ip4": StaticSelect2(),
             "primary_ip6": StaticSelect2(),
+            "software_version": StaticSelect2(),
```

Comment:
Why are these needed? Shouldn't we just be explicitly using `DynamicModelChoiceField` for these three fields instead?